client/shop.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Shop</title>
+  <meta name="robots" content="noindex">
+  <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
+  <script src="/js/utils.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css">
+  <link rel="stylesheet" href="/css/teststyle.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
+</head>
+<body>
+  <!------- header section ------->
+  <div class="header">
+    <div class="container">
+      <div class="navbar">
```

Comment:
take a look at `public/js/navbar.js`, it fills in the navbar links for the current season rather than doing static navbar changes. you should probably update this with a merch page that links to `auroraesports.org/{season}/Merch`